cmd/kubeadm/app/phases/certs/certs.go

```diff
@@ -396,6 +396,11 @@ func validateSignedCert(l certKeyLocation) error {
 		return fmt.Errorf("failure loading certificate authority for %s: %v", l.uxName, err)
 	}
 
+	return validateSignedCertWithCA(l, caCert)
+}
+
+// validateSignedCertWithCA tries to load a certificate and validate it with the given caCert
+func validateSignedCertWithCA(l certKeyLocation, caCert *x509.Certificate) error {
```

Comment:
not urgent, but might be good to get test cases for this function.

Reply:
It's just a slight generalisation of the existing validateSignedCert so I didn't bother.